portfolio-styling.css

```diff
@@ -0,0 +1,177 @@
+/* Basic stuff for reset, site styles and text sizing */
+
+* {
+	box-sizing: border-box;
+
+}
+
+h1, h2, h3, h4, h5, h6, p, a {
+	margin: 0;
+	padding: 0;
+	text-decoration: none;
+	font-weight: normal;
+}
+
+html {
+	font-size: 15px;
+	font-family: "Courier New";
+}
+
+h1 {
+	white-space: nowrap;
+	display: none;
+}
+
+h2 {
+	font-size: 2.4 rem;
+	width: 100%;
+	text-align: center;
+}
+
+h2.section-title {
+	visibility: hidden;
+	margin: 20px 0 20px 0;
+}
+
+h3 {
+	font-size: 1.8 rem;
+	width: 100%;
+	margin: 5% 0 5% 0;
+	text-align: center;
+}
+
+p {
+	font-size: 1.2 rem;
+	padding: 8%;
```

Comment:
using % for padding means it will be based on the size of the parent  - for `<p>` it could be placed anywhere, so I'd use pixels instead for padding